release/models/platform/openconfig-platform.yang

```diff
@@ -531,6 +539,36 @@ module openconfig-platform {
         this reports the role of the component.";
     }
 
+    container last-poweroff-reason {
+      description
+        "Records last power-off reason for a component.";
+
+      uses oc-platform-types:component-last-power-off-reason-trigger;
+    }
+
+    leaf last-poweroff-time {
+      type oc-types:timeticks64;
+      units "nanoseconds";
+      description
+        "This reports the time of the last shutdown of the component. 
+         The value is the timestamp in nanoseconds relative to the Unix Epoch (Jan 1, 1970 00:00:00 UTC). 
+         If a system is rebooted and after the configuration is loaded, say the controller-card is identified 
+         to be shutdown in configuration, the card should be powered-off and 'last-power-off-time' 
+         leaf should be recorded.";
+    }
```

Comment:
keep line length to < 70 characters per https://github.com/openconfig/public/blob/master/doc/openconfig_style_guide.md#line-length

Reply:
How about now? Wondering if it is okay to keep the paths as is?

Reply:
Hi, I don't see any changes in the line length?  Did you push your change?

Reply:
Hopefully now it shows up?